src/components/AccessControlNew.tsx

```diff
@@ -220,12 +260,50 @@ const AccessControlNew = () => {
                                                     <Input placeholder="Add rule name..." ref={inputNameRef} onPressEnter={() => toggleEditName(false)} onBlur={() => toggleEditName(false)} autoComplete="off"/>
                                                 </Form.Item>
                                             )}
+                                            { !editDescription ? (
+                                                <div className={"access-control input-text ant-drawer-subtitle"} onClick={() => toggleEditDescription(true)}>{formRule.Description && formRule.Description.trim() !== "" ? formRule.Description : 'Add description...'}</div>
+                                            ) : (
+                                                <Form.Item
+                                                    name="Description"
+                                                    label={null}
+                                                    style={{marginTop: 24}}
```

Comment:
Codacy has a fix for the issue: [Replace `····················································style={{marginTop:·24` with `··························style={{·marginTop:·24·`](https://app.codacy.com/gh/netbirdio/dashboard/pullRequest?prid=9560373)
```suggestion
                          style={{ marginTop: 24 }}
```